server/app/views/HtmlBundle.java

```diff
@@ -191,6 +192,9 @@ private FooterTag renderFooter() {
     }
     footerTag.with(viewUtils.makeLocalJsTag(jsBundle.getJsPath()));
 
+    // Add USWDS scripts
+    footerTag.with(viewUtils.makeLocalJsTag(USWDS_FILEPATH));
```

Comment:
question (non-blocking): what's the difference between `dist/uswds.bundle` and `javascripts/uswds/uswds-init.min` and why are they added the the DOM in different places?

Reply:
`uswds-init.min` is a small JavaScript file (less than 1 KB minified) that helps the browser know if the USWDS JavaScript library is loading properly.  `uswds.bundle` is the main uswds JavaScript file.  They are in two different places in the DOM because I followed the recommendations here:  [uswds docs](https://designsystem.digital.gov/documentation/developers/#using-uswds-css-and-javascript-in-your-project-2).

Note that I didn't opt to bundle `uswds-init.min` with Webpack because it's already bundled and is very small.  I figured that it was unlikely to change, so copied the minified file directly into our project.  Does that seem reasonable to you?

Reply:
Yes, thank you for the explanation